pinot-tools/src/main/java/com/linkedin/pinot/tools/PinotZKChanger.java

```diff
@@ -92,7 +96,7 @@ public int isStable(String tableName) {
     return numDiff;
   }
 
-  protected void waitForStable(String resourceName)
+  public void waitForStable(String resourceName)
```

Comment:
would be nice if you can add a javadoc on what this method does. I think it waits for EV==IS. Does it wait indefinitely? If so, is there a way to get progress? Or, does it matter?